app/views/news_tematica/news_tematicas/_bloque_mas_leidos.html.haml

```diff
@@ -9,5 +9,4 @@
         %h5.titulo= link_to_con_estadisticas masleido.titulo, masleido.contenido_dominio_link, @news_tematica
         %p.resumen
           = raw masleido.descripcion
-          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>', masleido.contenido_dominio_link, @news_tematica,
-            class: 'leer_mas')
+          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>'.html_safe, masleido.contenido_dominio_link, @news_tematica, class: 'leer_mas')
```

Comment:
I think that using css `nowrap` should be great, but styles are applied on main_apps, not in here. Also this is going to be sent by email and I don't know if is a good idea to use inline styling...